controllers/operands/kubevirt_test.go

```diff
@@ -174,6 +174,8 @@ var _ = Describe("KubeVirt Operand", func() {
 
 		defer os.Unsetenv(smbiosEnvName)
 		defer os.Unsetenv(machineTypeEnvName)
+		defer os.Unsetenv(amd64MachineTypeEnvName)
+		defer os.Unsetenv(arm64MachineTypeEnvName)
```

Comment:
It's already 5 of them. WDYT about adding a function to unset all of them and all it from a single `defer`?